### PR TITLE
MAT-8973 added updated and drafted logs

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-8973](https://jira.cms.gov/browse/MAT-8973)
(Optional) Related Tickets:

### Summary
added updated and drafted logs

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included and Code coverage is verified.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/cqllibraryservice/services/ActionLogService.java [hunk 1]
```diff
@@ -34,7 +33,7 @@ public boolean logAction(
             .actionType(actionType)
             .performedBy(userId)
             .performedAt(Instant.now())
-            .additionalActionMessage(Arrays.toString(additionalActionMessage))
+            .additionalActionMessage(String.join(", ", additionalActionMessage))
```
Comment (nmorasb) [inline src/main/java/gov/cms/madie/cqllibraryservice/services/ActionLogService.java]: will this result in NPE if we have anywhere this is called with null `additionalActionMessage`?

Comment (sb-prateekkeerthi) [reply src/main/java/gov/cms/madie/cqllibraryservice/services/ActionLogService.java]: for now if there is no additional message we don't explicitly send as null for additional message so based on that I have not added the NPE check

right now if there is no additional message we just send: 
  actionLogService.logAction(id, ActionType, username, "db name"); --> here if you see we are not providing additional message so in this case additional message will be [] and converting Arrays.toString will make it ""
  
 but yeah if we explicitly provide it as null as a parameter then yeah definitely we will get NPE. I can add a condition to explicitly to solve this if by any chance if we decide to send null as additional message

**Review by nmorasb — APPROVED** [review]
